WebOps Dashboard: Add a Live store ID section (#1293)
vtexdocs/dev-portal-content: docs/faststore/docs/getting-started/1-onboarding/dashboard.mdx

| :-------------------- | :----------------------------------------------------------------------------------------------- |
| **Live Store**        | URL for your production environment, accessible to shoppers.                                     |
| **Repository**        | GitHub repository associated with the project.                                                   |
| **Deploy**            | Commit for the current deployment, including the date, time, and who made it.                        |
| **CMS Provider**      | Link to [Headless CMS](https://developers.vtex.com/docs/guides/faststore/headless-cms-overview). |
| **CMS Project**       | Link to the FastStore project in Headless CMS.                                                   |

#### Live store ID

> After the onboarding in the WebOps app, you can access your store through a Live Store URL. You can find the Live Store URL in the VTEX Admin by accessing **Storefront > FastStore WebOps**. In the [**Production Overview**](https://developers.vtex.com/docs/guides/faststore/1-onboarding-dashboard#production-overview) card, locate the **Live Store** URL.

Reviewer: ```suggestion
After onboarding with the WebOps app, you can access your store through a live store URL. To find the live store URL, access **Storefront > FastStore WebOps** in the VTEX Admin. In the [**Production Overview**](https://developers.vtex.com/docs/guides/faststore/1-onboarding-dashboard#production-overview) card, locate the **Live Store** URL.
```

The live store URL includes a `liveStoreId`, a unique identifier for your store. It combines your store account name and a live store ID number, for example: `https://mystore-clwjc7fw200ay9qau1smn8i4l-21vm59f3q.b.vtex.app`. Let's break down the example to understand the `liveStoreId`:

- `mystore-clwjc7fw200ay9qau1smn8i4l-21vm59f3q.b`: the `liveStoreId`.
- `mystore`: the account name.
- `clwjc7fw200ay9qau1smn8i4l-21vm59f3q.b`: the live store ID number.

> The live store ID also helps you [set up the Preview URL in the Headless CMS](https://developers.vtex.com/docs/guides/faststore/headless-cms-1-configuring-the-vtex-account#step-2-configuring-the-headless-cms) so you can preview a page's changes.

Reviewer: ```suggestion
> The live store ID also helps you [set up the Preview URL in the Headless CMS](https://developers.vtex.com/docs/guides/faststore/headless-cms-1-configuring-the-vtex-account#step-2-configuring-the-headless-cms), enabling you to preview changes to your pages via the Headless CMS.
```
